Tests/UnitTests/ReaderTests/Data/ImageReaderSimpleRegression_map.txt

```diff
@@ -0,0 +1,4 @@
+images\11.png	1	1
```

Comment:
Please add an end-to-end test that uses NaN valued labels for training and for testing. See existing E2E tests as examples.